change/@ni-nimble-components-2feae987-64c1-4869-82bb-459373ad5067.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Upgrade to Angular 13.",
```

Same rationale as described above on the blazor package
```suggestion
  "comment": "Update `@ni/eslint-config-javascript` dependency version.",
```